Stripe/STPAddCardViewController.m

```diff
@@ -524,8 +545,27 @@ - (void)switchTableViewCell:(STPSwitchTableViewCell *)cell didToggleSwitch:(BOOL
 
 #pragma mark - UITableView
 
+#if DEBUG
+- (void)setForceEnableRememberMeForTesting:(BOOL)forceEnableRememberMeForTesting {
```

Comment:
this looks fine, can we just add a comment explaining that this is used for UI tests?